index.html

```diff
@@ -151,12 +152,14 @@ <h1>Contact me</h1>
     <p class="second-text">If you have an application you are interested in developing, a feature that you need built
       or a project that needs coding.
       I’d love to help with it!</p>
-    <div class="form">
-      <input type="text" placeholder="Lorem Ipsum">
-      <input type="text" placeholder="Loremipsum@gmail.com">
-      <textarea placeholder="Write your message here..." rows="10"></textarea>
-      <button>Get in touch</button>
-    </div>
+    <form class="form" action="https://formspree.io/f/meqwlynw" method="post">
+      <input type="text" placeholder="Lorem Ipsum" required maxlength="30">
+      <input type="text" placeholder="Loremipsum@gmail.com" required>
```

Comment:
- [ ] Since this input is meant for holding emails values I think it's way better if the `type` sets to email instead of text 💪🏼 
- [ ] Ensure that this input has a meaningful placeholder 👍🏼 💯